woof-distro/x86_64/void/voidpup64/rootfs-petbuilds/busybox-void

```diff
@@ -0,0 +1 @@
+busybox
```

Comment:
Just change to `busybox` and drop this symlink

Reply:
> Just change to `busybox` and drop this symlink

I tried that and busybox fails to petbuild compile unless there is already a .pet of busybox included in the build..... I forgot to capture the error ............. its probably some quirk of the void lib architecture (maybe)....

Reply:
Yes - because another version of busybox (from .pet) is already in the build